.eslintrc.cjs

```diff
@@ -57,6 +57,14 @@ module.exports = {
       'error',
       { checksVoidReturn: false },
     ],
+    'import/order': [
```

Comment:
There are a couple Prettier plugins that do similar things, but I didn't find a way to have the same config in both ESLint and Prettier :/